docker-compose.yml

```diff
@@ -82,22 +82,22 @@ services:
             - tempo:/tmp/tempo
             - ./docker/grafana/tempo.yml:/etc/tempo.yml:ro
         ports:
-            - 3200:3200    # tempo
-            - 9411:9411    # zipkin
+            - "3200:3200"    # tempo
+            - "9411:9411"    # zipkin
     loki:
         container_name: loki
         image: grafana/loki:3.0.0 # https://hub.docker.com/r/grafana/loki/tags and https://github.com/grafana/loki/releases
         extra_hosts: ['host.docker.internal:host-gateway']
         command: ['-config.file=/etc/loki/local-config.yaml']
         ports:
-            - 3100:3100
+            - "3100:3100"
     maildev:
         container_name: maildev
         image: maildev/maildev:2.1.0 # https://hub.docker.com/r/maildev/maildev/tags
         extra_hosts: [ 'host.docker.internal:host-gateway' ]
         ports:
-            - 3001:1080
-            - 25:1025
+            - "3001:1080"
+            - "25:1025"
```

Comment:
If my memory serves me well, not using `"` is intentional here since Docker Compose has/had issues with it for lower port numbers.
Unless this was fixed and you have a reason why quotes should be here, I would prefer not to include them.

Reply:
Created #16



> HOST:CONTAINER should always be specified as a (quoted) string, to avoid conflicts with yaml base-60 float

https://docs.docker.com/compose/compose-file/05-services/#ports